x/profiles/types/error.go

```diff
@@ -7,16 +7,74 @@ import (
 )
 
 var (
-	ErrInvalidVersion      = sdkerrors.Register(ModuleName, 1, "invalid version")
+	// ErrInvalidVersion is returned if a wrong version of IBC channel is used
+	ErrInvalidVersion = sdkerrors.Register(ModuleName, 1, "invalid version")
+
+	// ErrMaxProfilesChannels is returned if the channel sequence exceed the max allowed
 	ErrMaxProfilesChannels = sdkerrors.Register(ModuleName, 2, "max profiles channels")
 
-	ErrProfileNotFound = sdkerrors.Register(ModuleName, 10, "profile not found")
+	// ErrProfileNotFound is returned when a profile doesn't exist
+	ErrProfileNotFound = sdkerrors.Register(ModuleName, 3, "profile not found")
+
+	// ErrProfileAlreadyCreated is returned when a profile has already been created
+	ErrProfileAlreadyCreated = sdkerrors.Register(ModuleName, 4, "profile already created")
```

Comment:
I think it might be better to rename this to something like `ErrDuplicatedDTag` if it's used only in that case